docs/rules/README.md

```diff
@@ -231,6 +230,7 @@ These rules are only relevant to ES6 environments.
 * [no-const-assign](no-const-assign.md) - disallow modifying variables that are declared using `const` (recommended)
 * [no-dupe-class-members](no-dupe-class-members.md) - disallow duplicate name in class members (recommended)
 * [no-new-symbol](no-new-symbol.md) - disallow use of the `new` operator with the `Symbol` object (recommended)
+* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified modules when loaded by `import` declaration [see http://www.ecma-international.org/ecma-262/6.0/index.html#sec-imports and https://github.com/eslint/eslint/blob/master/lib/rules/no-restricted-imports.js#L22]
```

Comment:
@brettz9 uh oh 😱 my bad to include the cross-references in square brackets. which were for your benefit in a way that looked as if they belonged in the list